LandSandBoat/server
# Use Task System to schedule Conquest tasks

**_I affirm:_**

## What does this pull request do?

- Conquest system no longer running two separate instances
- Use task system to schedule conquest system tasks

## Steps to test these changes

Run conquest updates. Make sure conquest map is consistent. Check all zones and make sure there are no crashes.
